### PR TITLE
Add support for shutdown of virtual controller

### PR DESCRIPTION
**Pull request by svt-mgawade**
Signed-off-by: Mitali Gawade <mitali.gawade@hpe.com>

### Description
Add support for shutdown of virtual controller

### Issues Resolved
None

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: simplivity/resources/hosts.py [hunk 1]
```diff
@@ -180,3 +180,22 @@ def get_virtual_controller_shutdown_status(self):
...
+
+        Args:
+          ha_wait: An indicator to show if the user wants to shut down the Virtual Controller safely or forcefully.
+            Valid values:
+              True: Virtual Controller waits for the virtual machines to reach HA compliance before shutting down.
+              False: Virtual Controller forced to shut down without waiting for HA compliance.
+          timeout: Time out for the request in seconds.
+
+        Returns:
+            object: status object.
```
Comment (svt-kmcdaniel) [inline simplivity/resources/hosts.py]: This isn't an object, it is just a python `dict`

Comment (svt-kmcdaniel) [reply simplivity/resources/hosts.py]: Please check the feedback on the cancel shutdown as some of them apply here.

Comment (svt-kmcdaniel) [reply simplivity/resources/hosts.py]: Should we just make this a string

File: simplivity/resources/hosts.py [hunk 2]
```diff
@@ -180,3 +180,22 @@ def get_virtual_controller_shutdown_status(self):
...
+        """
+        method_url = "{}/{}/shutdown_virtual_controller".format(URL, self.data["id"])
+        data = {"ha_wait": ha_wait}
+
+        response = self._client.do_post(method_url, data, timeout)
+        return response
```
Comment (svt-kmcdaniel) [inline simplivity/resources/hosts.py]: should this be a string?

**Review by svt-kmcdaniel — CHANGES_REQUESTED** [review]

File: simplivity/resources/hosts.py (hunk 1 above)
Comment (svt-mgawade) [reply simplivity/resources/hosts.py]: Updated to code suggested

File: simplivity/resources/hosts.py (hunk 2 above)
Comment (svt-mgawade) [reply simplivity/resources/hosts.py]: Updated to code suggested

**Review by sijeesh — APPROVED** [review]
LGTM

**Review by svt-kmcdaniel — APPROVED** [review]

**Pull request merged**